Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT OR BSD-3-Clause"
 name = "safe-nd"
 readme = "README.md"
 repository = "https://github.com/maidsafe/safe-nd"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
We need to update `Changelog.md` as well. Also, version changes usually happen in their own PRs (i.e. without any other changes).